### PR TITLE
Add more controller permissions in recommended-inline-policy.json

### PR DESCRIPTION
**Pull request by zijun726911**
- Add more controller permissions in recommended-inline-policy.json
- Fix e2e test code compiling error

### Tests

New permissions could run access_log_policy_test.go success

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -66,6 +66,7 @@ vet: ## Vet the code and dependencies
 		if [ "${CI}" = true ]; then\
 			exit 1;\
 		fi;}
+	cd test && go vet ./...
```
Comment (mikhail-aws) [inline Makefile:69]: go vet ./... should include all sub directories, it's not module based afaik

**Review by mikhail-aws — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (zijun726911) [reply Makefile:69]: From my test, seems the root dir `go vet ./...` cannot detect e2etest folder compiling error

Comment (mikhail-aws) [reply Makefile:69]: what about go vet ./test/... ?

Comment (zijun726911) [reply Makefile:69]: I just tried many `go vet` commands, seems only `cd test && go vet ./...` can work

**Review by mikhail-aws — APPROVED** [review]

**Pull request merged**